presentation/common/src/presentation-common/PresentationRpcInterface.ts

```diff
@@ -169,37 +169,55 @@ export class PresentationRpcInterface extends RpcInterface {
   ===========================================================================================*/
 
   public async getNodesCount(_token: IModelRpcProps, _options: HierarchyRpcRequestOptions): PresentationRpcResponse<number> { return this.forward(arguments); }
+
+  @RpcOperation.setPolicy({ allowResponseCompression: true })
   public async getPagedNodes(_token: IModelRpcProps, _options: Paged<HierarchyRpcRequestOptions>): PresentationRpcResponse<PagedResponse<NodeJSON>> { return this.forward(arguments); }
 
   // TODO: add paged version of this (#387280)
+  @RpcOperation.setPolicy({ allowResponseCompression: true })
   public async getNodePaths(_token: IModelRpcProps, _options: FilterByInstancePathsHierarchyRpcRequestOptions): PresentationRpcResponse<NodePathElementJSON[]> { return this.forward(arguments); }
+
   // TODO: add paged version of this (#387280)
+  @RpcOperation.setPolicy({ allowResponseCompression: true })
   public async getFilteredNodePaths(_token: IModelRpcProps, _options: FilterByTextHierarchyRpcRequestOptions): PresentationRpcResponse<NodePathElementJSON[]> { return this.forward(arguments); }
 
   /** @beta */
+  @RpcOperation.setPolicy({ allowResponseCompression: true })
   public async getContentSources(_token: IModelRpcProps, _options: ContentSourcesRpcRequestOptions): PresentationRpcResponse<ContentSourcesRpcResult> { return this.forward(arguments); }
 
+  @RpcOperation.setPolicy({ allowResponseCompression: true })
```

Comment:
@jason-crow, you probably want to enable this for the element properties RPC that you have in DR backend.